libsigner/src/runloop.rs

```diff
@@ -63,10 +63,11 @@ pub trait SignerRunLoop<R> {
     fn main_loop<EVST: EventStopSignaler>(
         &mut self,
         event_recv: Receiver<StackerDBChunksEvent>,
+        command_recv: Receiver<CMD>,
         mut event_stop_signaler: EVST,
     ) -> Option<R> {
         loop {
-            let poll_timeout = self.get_event_timeout();
+            let poll_timeout = Duration::from_millis(128); //self.get_event_timeout();
```

Comment:
I think this actually can be fixed in this PR. If you just update the duration set in the timeout in the config file to be 128 millisecs, it should have the same effect. It is an optional that defaults to 5 seconds right now. however, you could add it as an option in the build_singer_config_tomls function or just chane the EVENT_TIMEOUT_SECS default.